todo-app/src/App.js

```diff
@@ -1,10 +1,15 @@
 import React, { Component } from 'react';
+import "./style.css"
+import Header from "./components/header.js"
```

Comment:
These component files should be named the same as the components themselves, please capitalise.